src/app/core/navbar/navbar.component.html

```diff
@@ -7,6 +7,11 @@
     <a mat-list-item routerLink="/" routerLinkActive="active" [routerLinkActiveOptions]="{exact: true}">
         <mat-icon>home</mat-icon><p data-cy="Dashboard"></p><span i18n="navlink">Dashboard</span>
     </a>
+    <a mat-list-item routerLink="/ieee8021x" routerLinkActive="active">
+        <mat-icon>security</mat-icon>
+        <span i18n="navlink">IEEE 8021.x</span>
```

Comment:
Small typo '8021.x', I'd make a PR but didn't want to mess up whatever you got going on.

Reply:
resolved